test/Flick/AnimatedCarousalPage.xaml.cs

```diff
@@ -49,6 +57,76 @@ public sealed partial class AnimatedCarousalPage : Page
         public AnimatedCarousalPage()
         {
             this.InitializeComponent();
+
+            ElementCompositionPreview.GetElementVisual(sv).Clip = ElementCompositionPreview.GetScrollViewerManipulationPropertySet(sv).Compositor.CreateInsetClip();
+        }
+
+        private object selectedItem = null;
+
+        public object SelectedItem
+        {
+            get
+            {
+                return selectedItem;
+            }
+            set
+            {
+                selectedItem = value;
+            }
+        }
```

Comment:
can just be 
public object SelectedItem { get;set;}

Reply:
Nope. Users of this control will need to be able to query the control's selected item.

Reply:
public object SelectedItem { get;set;}

---
In reply to: [287158964](https://github.com/microsoft/microsoft-ui-xaml/pull/743#discussion_r287158964) [](ancestors = 287158964)